packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -251,6 +243,86 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
   return [...restOfSentryVitePlugins, customPlugin];
 }
 
+/**
+ * Whether the user enabled (true, 'hidden', 'inline') or disabled (false) source maps
+ */
+export type UserSourceMapSetting = 'enabled' | 'disabled' | 'unset' | undefined;
+
+/** There are 3 ways to set up source maps (https://github.com/getsentry/sentry-javascript/issues/13993)
```

Comment:
l: just to be a bit more specific:
```suggestion
/** There are 3 ways to set up source map generation (https://github.com/getsentry/sentry-javascript/issues/13993)
```